samples/postman/Lighthouse OAuth Token.postman_collection.json

```diff
@@ -0,0 +1,185 @@
alth API aud value:\nhttps://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token\nb. token_endpoint (Collection variable)\nExample Patient Health API tokenUrl value:\nhttps://sandbox-api.va.gov/oauth2/health/system/v1/token\nc. clientId (Collection variable)- This value is obtained from signing up for sandbox access on this link:\nhttps://developer.va.gov/explore/api/patient-health/sandbox-access\nExample clientId value:\nYour CCG clientId\n\nd. privatePem (Collection variable)\nExample privatePem value:\n-----BEGIN RSA PRIVATE KEY-----\nYour private pem\n-----END RSA PRIVATE KEY-----\n\n(5). Next under the \"Lighthouse OAuth Token\" click on the actual POST call you will make - \"POST Client Credentials Example\"\n\n(6). You will see that the HTTP url is the {token_endpoint} Collection variable that has already been configured.\n\n(7). Then click on \"Body\" which is located in the middle of the Auth type columns. You should see all values pre-populated and using the x-www-form-urlencoded type. The grant_type and client_assertion_type values should not change as this is a CCG call. The client_assertion should also remain the same as it will be populated with the client_assertion Collection variable. The two configurable values will be the scope and launch based off of the API and call you choose to make. Follow the guidance provided at https://developer.va.gov/explore/api/patient-health/client-credentials to obtain the necessary values for the Patient Health API. For other APIs, consult the Lighthouse CCG API page related to your desired interaction for accurate API-specific collection variable values.\n\na. scope\nCurrently the default value is set to \"launch\" which is acceptable for Patient Health API but can be changed to include:\nExample Patient Health API scope value:\nlaunch system/AllergyIntolerance.read system/Appointment.read system/Condition.read system/Coverage.read system/Coverage.write system/DiagnosticReport.read system/Immunization.read system/Location.read system/Medication.read system/MedicationOrder.read system/Observation.read system/Organization.read system/Patient.read\n\nb. launch\nIn some cases launch will not be required for certain APIs. Follow the documentation to see which APIs require launch. In the cases where launch isn't required, uncheck the \"launch\" box and the default scope value will also need to change accordingly. For Patient Health API launch is required to be an encoded base64.\nExample Value:\neyJwYXRpZW50IjoiMTAwMDcyMDEwMFYyNzEzODcifQ==\n\nRun:\n\nThis step will generate a JWT assertion and return an OAuth access token. Users can view the JWT assertion in the console after running \"Client Credentials Example\".\n\n(1). If not already there, click on “Client Credentials Example” in the left column of Postman.\n\n(2). Click \"Send\" on the top right corner.\n\nNote:\nIf all values were configured correctly, you should see an access_token and other fields corresponding to the API call you made.",
```

Comment:
very good, thorough and clear.. I would just take out doc link, the docs will link to this and it doesn't need to be api specific 

> Follow the guidance provided at https://developer.va.gov/explore/api/patient-health/client-credentials to obtain the necessary collection variable values for the Patient Health API

This can just become a link to the developer portal. (https://developer.va.gov/explore)